src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -2837,6 +2847,16 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         e.DragUIOverride().IsGlyphVisible(false);
     }
 
+    // Method description:
+    // - Checks if the uri is valid and sends an event if so
+    // Arguments:
+    // - The uri
+    void TermControl::_HyperlinkHandler(std::wstring uri)
+    {

```

Comment:
prefer wrapping this guy in a try/catch_log just so we can't take down the terminal when somebody clicks on an invalid URI (which fails to parse in the Uri constructor)